app/javascript/src/AccessTokens/components/ExpirationDatePicker.scss

```diff
@@ -0,0 +1,14 @@
+@import '~@patternfly/patternfly/patternfly-addons.css';
```

Comment:
This looks like something we would import at an upper level, like a base css file. I am not sure what it is for (it's used only in dev portal). I'm worried that it's going to be duplicated someday, and unfortunately there's no way to dedupe @imports from individual scss files.

Reply:
This is to load the `pf-u-mt-md` class. I could include it from somewhere else, do you have a suggestion?

Reply:
The best way to do this would be to create a new scss pack that imports patternfly-addons alone, then import the pack alongside expiration_date_picker.ts. HOWEVER, it's not something we have to do right now. Let's keep it like this and improve only if it's ever necessary.